activerecord/test/cases/associations/has_many_associations_test.rb

```diff
@@ -244,8 +244,9 @@ def test_create_from_association_with_nil_values_should_work
   def test_do_not_call_callbacks_for_delete_all
     car = Car.create(:name => 'honda')
     car.funky_bulbs.create!
+    assert_equal 1, car.funky_bulbs.count
     assert_nothing_raised { car.reload.funky_bulbs.delete_all }
-    assert_equal 0, Bulb.count, "bulbs should have been deleted using :delete_all strategy"
+    assert_equal 0, car.funky_bulbs.count, "bulbs should have been deleted using :delete_all strategy"
```

Comment:
This might be because of random failure. The change is actually correct. If any other file uses `bulbs` fixtures then `Bulb.count` is non-zero before this test, which causes this test to fail.